.gitignore

```diff
@@ -5,6 +5,7 @@
 /templates/typescript/cloudevents/build
 /templates/typescript/http/build
 /templates/go/cloudevents/go.sum
+/templates/go/.instanced-cloudevents/go.sum
```

Comment:
Upon closer look, I should have said to just check in the sumfile.  Line 7 here will be removed when we replace the middleware with `func-go`